src/DotNetWorker.Core/Context/FunctionContextLoggerExtensions.cs

```diff
@@ -30,8 +30,8 @@ public static ILogger<T> GetLogger<T>(this FunctionContext context)
         public static ILogger GetLogger(this FunctionContext context, string categoryName)
         {
             return context.InstanceServices
-                   .GetService<ILoggerFactory>()!
-                   .CreateLogger(categoryName);
+                    .GetService<ILoggerFactory>()!
+                    .CreateLogger(Logging.LogCategories.CreateFunctionUserCategory(categoryName));
```

Comment:
This doesn't seem like something we'd want to do... it'd change the category that someone explicitly passed into us.